.github/workflows/web.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Web
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - uses: shogo82148/actions-setup-redis@v1
+      with:
+        redis-version: '5.x'
+    - name: Install node packages
+      run: npm ci
+    - name: Lint
+      run: node_modules/.bin/tslint --project tsconfig.json --config tslint.json
+    - name: Test
```

Comment:
Should we test after building?

Reply:
Building takes longer and tests usually broke. I put the tests first so that the waiting time to expose the problems can be shorter.